### PR TITLE
Fix: project-background-color group as startup base layer

### PR DESCRIPTION
**Pull request by rldhont**
Completes https://github.com/3liz/lizmap-web-client/pull/4435 by fixing and testing that `project-background-color` group will be the startup base layer

### CODE REVIEW

**Comment by rldhont** [comment]
Linked to https://github.com/3liz/lizmap-web-client/pull/4367 and https://github.com/3liz/lizmap-web-client/pull/4319

**Comment by rldhont** [comment]
@Gustry I have fixed e2e tests by moving WMS/WMTS layers from demo.lizmap.com to liz.lizmap.com

File: tests/js-units/node/config/baselayer.test.js [hunk 1]
```diff
@@ -834,7 +834,7 @@ describe('BaseLayersConfig', function () {
             "format": "image/png;%20mode%3D8bit",
             "layers": "commune",
             "styles": "d%C3%A9faut",
-            "url": "https://demo.lizmap.com/lizmap/index.php/lizmap/service?repository=miscellaneous&project=flatgeobuf&VERSION=1.3.0"
+            "url": "https://liz.lizmap.com/tests/index.php/lizmap/service?repository=miscellaneous&project=flatgeobuf&VERSION=1.3.0"
```
Comment (Gustry) [inline tests/js-units/node/config/baselayer.test.js:837]: Arf, thanks.
Yesterday, as I was updating the demo website, I removed the `cypress` folder. Thanks for the update to the liz server.

**Review by Gustry — APPROVED** [review]

**Comment by Gustry** [comment]
Thanks for the fix.
Unfortunetaly, it was enough to fix E2E tests 

```
   2) [end2end] › theme.spec.js:92:9 › Theme › must display theme4 when selected ────────────────────

    Error: Timed out 5000ms waiting for expect(locator).toHaveValue(expected)

    Locator: locator('lizmap-base-layers select')
    Expected string: "OpenStreetMap"
    Received string: "project-background-color"
```

on PR #4367
https://github.com/3liz/lizmap-web-client/actions/runs/9513791364/job/26224604488?pr=4367

Feel free to include temporary this commit to try, or to push on my branch.

**Pull request merged**